template/src/main/java/com/asyncapi/model/$$message$$.java

```diff
@@ -0,0 +1,14 @@
+package com.asyncapi.model;
+
+public class {{messageName | camelCase | upperFirst}} {
```

Comment:
You could use [Lombok](https://projectlombok.org/) here and reduce the code you need to template.

Reply:
I do not want to start a holy war about usage of Lombok 😃